src/code.cloudfoundry.org/gorouter/config/config.go

```diff
@@ -38,7 +39,8 @@ const (
 )
 
 var (
-	LoadBalancingStrategies         = []string{LOAD_BALANCE_RR, LOAD_BALANCE_LC}
+	GlobalLoadBalancingAlgorithms   = []string{LOAD_BALANCE_RR, LOAD_BALANCE_LC}             // These strategies can be set globally via routing-release config
+	LoadBalancingStrategies         = append(GlobalLoadBalancingAlgorithms, LOAD_BALANCE_HB) // These strategies can be set for individual routes (per-route options)
```

Comment:
I would prefer two explicit lists:

```suggestion
	GlobalLoadBalancingAlgorithms   = []string{LOAD_BALANCE_RR, LOAD_BALANCE_LC}
	LoadBalancingStrategies         = []string{LOAD_BALANCE_RR, LOAD_BALANCE_LC, LOAD_BALANCE_HB}
```

tying together unrelated lists has bitten us in the pasts with the error checking. When adding a new algorithm the developer should make the conscious choice to add it to either, or, or both.

(I also think the names speak for themselves, removed the comment in the suggestion)

Reply:
I like your idea.